spec/requests/user_spec.rb

```diff
@@ -16,6 +16,7 @@
     {
       email: "email@example.com",
       email_verified: true,
```

Comment:
given this is a one-way-switch...
is there any utility in making this someting more like:
`email_verified_on: "2021-01-01"`, where `nil` means never verified?
Wonder if that is a better form of history than `true` or `false?

Whereas `has_unconfirmed_email` is a representation of present state?

Reply:
atch, but then we'd have to update `email_verified_at` and perhaps that's just another thing that could go wrong. Probably nevermind!

Reply:
Another option might be to have a three state thing:

```
email_state = :confirmed
email_state = :pending_reconfirmation
email_state = :pending_confirmation
```